### ISSUE TITLE
Revisit how we create slugs for new exhibits

### ISSUE THREAD

**Issue by ggeisler** [sul-dlss/exhibits#196]
Creating the exhibit slug from the title provided on the new exhibit form is a logical way to do it but it seems likely that when Exhibits is fully self-service many curators will not adjust the default slug when creating a new exhibit and we'll end up with exhibits with long, unfriendly URLs. 

A simple way to mitigate this a bit might be additional help text on the exhibit creation page to point out the benefits of a shorter slug.

A more involved but ultimately better solution would be to make it easier for at least super-admins to update an exhibit slug through the UI.

**Comment by ggeisler** [comment]
This might not be as important if we make a policy decision (as we seem to be leaning towards) that exhibits are only created by super-admins and the Spotlight service team.

**Comment by atz** [comment]
Yeah, the importance of that defaulted value is not emphasized enough.
